core/src/test/java/com/scalar/db/storage/dynamo/DynamoAdminTestBase.java

```diff
@@ -64,45 +63,66 @@
 import software.amazon.awssdk.services.dynamodb.model.UpdateContinuousBackupsRequest;
 import software.amazon.awssdk.services.dynamodb.model.UpdateTableRequest;
 
-public class DynamoAdminTest {
+/**
+ * Abstraction that defines unit tests for the {@link DynamoAdmin}. The class purpose is to be able
+ * to run the {@link DynamoAdmin} unit tests with different values for the {@link DynamoConfig},
+ * notably {@link DynamoConfig#NAMESPACE_PREFIX} and {@link DynamoConfig#TABLE_METADATA_NAMESPACE}
+ */
+public abstract class DynamoAdminTestBase {
```

Comment:
To run unit tests with different values of the `DynamoConfig` regarding the namespace prefix or the name of the metadata table, I moved all the tests to the abstract class `DynamoAdminTestBase` and added the inhereting classes : 
- DynamoAdminTestWithDefaultConfigTest
- DynamoAdminWithNamespacePrefixAndTableMetadataNamespaceConfigTest
- DynamoAdminWithNamespacePrefixConfigTest

I applied the same logic when adding tests for the different `XXStatementHandler` classes.

Reply:
Thank you for the refactoring! That really clean up the tests.